src/PaprController.sol

```diff
@@ -492,7 +492,7 @@ contract PaprController is
 
         uint256 max = _maxDebt(_vaultInfo[account][asset].count * oraclePrice, cachedTarget);
 
-        if (newDebt > max) revert IPaprController.ExceedsMaxDebt(newDebt, max);
+        if (!(newDebt < max)) revert IPaprController.ExceedsMaxDebt(newDebt, max);
```

Comment:
i don't exactly understand why we need to negative the > ? aren't these two identical?

Reply:
we want that it is strictly LESS than. So we want to revert if it is NOT LESS THAN. Currently it allows =

Reply:
got it